src/qibolab/compilers/default.py

```diff
@@ -24,6 +24,17 @@ def rz_rule(gate, platform):
     return PulseSequence(), {qubit: gate.parameters[0]}
 
 
+def gpi2_rule(gate, platform):
+    """Rule for GPI2."""
+    qubit = gate.target_qubits[0]
+    theta = gate.parameters[0]
+
+    sequence = PulseSequence()
+    pulse = platform.create_RX90_pulse(qubit, start=0, relative_phase=theta)
+    sequence.add(pulse)
+    return sequence, {qubit: gate.parameters[0]}
```

Comment:
I am not sure if GPI2 requires to add a virtual phase in the upcoming pulses, which is what this dictionary is for. I have the impression that you only need to put `theta` in the RX90 pulse phase, which you do, and there is no further virtual-Z rotation, but I am not sure at all so it is better if @igres26 can confirm this.

Reply:
I see, I don't think so. Lets wait for @igres26 feedback.

Reply:
I had a chat with @igres26 and there is no further virtual-Z rotation need. I've removed it in https://github.com/qiboteam/qibolab/pull/549/commits/7620487155e9443c8f0f9d57c1f79b25898b0f77.